week1/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
+    <title>help hamsick</title>
+  </head>
+
+  <body>
+    <header>
+      <div class="headerColorBox">
+      <h1>💛🍔 햄식이를 도와줘 🍔💛</h1>
+      </div>
+    </header>
+
+    <main>
+      <aside class="sidebar">
+        <section class="option_card">
+          <h2 class="optionTitle">다른거 먹자</h2>
+          <div class="option"> 
+            <p class="macdonalds">맥도날드</p>
+            <p class="burgerking">버거킹</p>
+          </div>
+        </section>
+      </aside>
+      <section class="burger">
```

Comment:
h2 테그를 사용한다면, 이 section구획에 맞는 제목을 h2테그로 적어둘 수 있을 것 같습니다! 
```suggestion
      <section class="burger">
         <h2>햄버거 리스트</h2>
```
h테그는 이런 느낌으로 사용해야 콘텐츠 전체의 구조를 파악하는데 있어서 의미있는 heading테그가 되는 것 같아요!

Reply:
h 테그의 층위 ? 에 대한 느낌을 주기 위해... 추가한 것이지.. 꼭 넣으라는 것은 아닙미다 !

Reply:
아아~~~~~! 위에서 살짝 뇌절왔는데 뭔지 딱알았어요